app/(gcforms)/[locale]/(form administration)/form-builder/[id]/settings/manage/notifications/Notifications.tsx

```diff
@@ -1,97 +1,128 @@
-    if (newNotificationsInterval === NotificationsInterval.OFF) {
+  const updateNotifications = useCallback(async () => {
+    const result = await saveNotificationsSettings(formId, sessionUser);
+    // TODO better way to do this?
+    if (result !== undefined && "error" in result) {
+      toast.error(updateNotificationsError);
+    } else {
       ga("form_notifications", {
         formId,
-        action: "disabled",
+        action: sessionUser?.enabled ? "enabled" : "disabled",
       });
+      toast.success(updateNotificationsSuccess);
     }
+  }, [formId, sessionUser, updateNotificationsError, updateNotificationsSuccess]);
 
-    const result = await updateNotificationsInterval(formId, newNotificationsInterval);
-
-    if (result && result.error) {
-      toast.error(updateNotificationsIntervalError);
-    } else {
-      setNotificationsInterval(newNotificationsInterval);
-      toast.success(updateNotificationsIntervalSuccess);
-    }
-  }, [
-    formId,
-    notificationValue,
-    updateNotificationsIntervalError,
-    updateNotificationsIntervalSuccess,
-    setNotificationsInterval,
-  ]);
+  // This is a published form with email delivery or a legacy form has no users.
+  // Don't show notifications settings.
+  if (getDeliveryOption() || !sessionUser) {
+    return null;
+  }
 
   return (
     <div className="mb-10" data-testid="form-notifications">
       <h2>{t("settings.notifications.title")}</h2>
-      <p className="mb-2 font-bold">{t("settings.notifications.description1")}</p>
-      <p className="mb-8">{t("settings.notifications.description2")}</p>
+      <p className="mb-2 font-bold">{t("settings.notifications.sessionUser.title")}</p>
+      <p className="mb-4">{t("settings.notifications.sessionUser.description")}</p>
       <div className="mb-4">
         <NotificationsToggle
-          isChecked={
-            isVault && notificationValue === String(NotificationsInterval.DAY) ? true : false
-          }
+          isChecked={sessionUser.enabled}
           toggleChecked={toggleChecked}
-          onLabel={t("settings.notifications.options.off")}
-          offLabel={t("settings.notifications.options.on")}
-          description={t("settings.notifications.optionsDescription")}
-          disabled={!isVault}
+          onLabel={t("settings.notifications.sessionUser.toggle.off")}
+          offLabel={t("settings.notifications.sessionUser.toggle.on")}
+          description={t("settings.notifications.sessionUser.toggle.title", {
+            email: sessionUser.email,
+          })}
         />
       </div>
-      <Button
-        dataTestId="form-notifications-save"
-        theme="secondary"
-        onClick={saveNotificationsValue}
-        disabled={!isVault}
-      >
+      {users && users.length > 0 && (
+        <>
+          <p className="mb-2 font-bold">{t("settings.notifications.usersList.title")}</p>
+          <ul className="m-0 mb-4 list-none p-0">
+            {users.map((notificationUser) => {
```

Comment:
Can we break this into it's own component and get it to manage it's own data --- it's just a list and looks like it can be separated from the toggle / update action.